Mathlib/Probability/Distributions/Uniform.lean

```diff
@@ -1,14 +1,20 @@
 /-
-Copyright (c) 2022 Devon Tuma. All rights reserved.
+Copyright (c) 2024 Josha Dekker. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Devon Tuma
+Authors: Josha Dekker, Devon Tuma, Kexing Ying
 -/
+import Mathlib.Probability.Notation
+import Mathlib.Probability.Density
 import Mathlib.Probability.ProbabilityMassFunction.Constructions
 
-#align_import probability.probability_mass_function.uniform from "leanprover-community/mathlib"@"e50b8c261b0a000b806ec0e1356b41945eda61f7"
-
 /-!
-# Uniform Probability Mass Functions
+This file defines two related notions of uniform distributions, which will be unified in the future.
```

Comment:
The first line of the docstring should usually be a `# Header` comment. Perhaps `Uniform distributions and probability mass functions`?

Reply:
Thank you, done!